### PR TITLE
gzip: add --keep example

### PR DESCRIPTION
**Pull request by scrouthtv**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

I don't know if this is a useful example, it can be found somewhat quickly by searching the manpage. Feel free to close if not.

### CODE REVIEW

File: pages/common/gzip.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 
 `gzip -d {{file.ext}}.gz`
 
+- Compress a file, keeping the original file:
+
+`gzip -k {{file.ext}}`
```
Comment (navarroaxel) [inline pages/common/gzip.md:16]: Could we use long arguments in this page?
```suggestion
`gzip --keep {{file.ext}}`
```

**Review by navarroaxel — COMMENTED** [review]
Could you update the arguments to the long version?

**Comment by scrouthtv** [comment]
Sure!

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @scrouthtv!

**Comment by scrouthtv** [comment]
:)

**Pull request merged**